src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -711,7 +711,10 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
             context.EnvironmentVariables["ASPNETCORE_URLS"] = appHostApplicationUrl;
             context.EnvironmentVariables["DOTNET_RESOURCE_SERVICE_ENDPOINT_URL"] = grpcEndpointUrl;
             context.EnvironmentVariables["DOTNET_DASHBOARD_OTLP_ENDPOINT_URL"] = otlpEndpointUrl;
-            context.EnvironmentVariables["ResourceServiceClient__AuthMode"] = "Unsecured"; // No auth in local dev experience
+
+            // No auth in local dev experience
+            context.EnvironmentVariables["ResourceServiceClient__AuthMode"] = "Unsecured";
```

Comment:
I know this is only tangential to the actual issue, but why do these not follow the UPPER_CASING of the other environment variables?

Reply:
IConfiguration keys are case-insensitive. I can update it if someone wants, but it isn't critical to be consistent.